.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,53 @@
+name: Build images from latest main branch
+
+on:
+  push:
+    branches:
+      - main
+jobs:
+  create-main-image:
+    name: env
+    runs-on: ubuntu-latest
+    environment: main
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Install node module dependencies
+        run: |
+          cd hapi
+          yarn install
+          cd ../webapp
+          yarn install
+          cd ..
+      - name: Build new images and push
+        run: |
+          make \
+            build-docker-images \
+            push-docker-images
+        env:
+          # global
+          DOCKER_USER: ${{ secrets.DOCKER_HUB_USER }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
+          HASURA_GRAPHQL_ADMIN_SECRET: ${{ secrets.HASURA_SECRET }}
+          POSTGRES_PASSWORD: ${{ secrets.POSTGRES_SECRET }}
+          EOS_PRIV_KEY: ${{ secrets.EOS_PRIV }}
+          MAIL_PASSWORD: ${{ secrets.MAIL_SECRET }}
+          NAMESPACE: lifebank
+          STAGE: dev
+          APP_NAME: lifebank
+          INGRESS_HOST: lifebank.io
+          INGRESS_GRAPHQL_HOST: graphql.lifebank.io
+
+          #webapp
+          PORT: 80
+          REACT_APP_EOS_API_HOST: lifebank.eosio.cr
+          REACT_APP_EOS_API_PORT: 443
+          REACT_APP_EOS_API_PROTOCOL: https
+          REACT_APP_EOS_CHAIN_ID: 3ba28e6d66769dd9723431d138aefe8cfb379f0f1a5885a17a2a826b71e31997
+          REACT_APP_HASURA_URL: http://localhost:8080/v1/graphql
```

Comment:
localhost?

Reply:
replace localhost with the container name